rules/aws_cloudtrail_rules/aws_iam_backdoor_user_keys.yml

```diff
@@ -0,0 +1,91 @@
+AnalysisType: rule
+Description: 'Detects AWS API key creation for a user by another user. Backdoored users can be used to obtain persistence in the AWS environment. '
```

Comment:
trailing space. Single quotes could also be removed ( quote removal seems like part of the changeset on rules/aws_cloudtrail_rules/aws_ec2_startup_script_change.yml  )